components/XOGame.js

```diff
@@ -1,14 +1,25 @@
 import styled from "styled-components";
-import {useState} from "react";
+import {useEffect, useState} from "react";
 
 const players = {
-
+  CPU: {
+    SYM: "O",
```

Comment:
```suggestion
    SYMBOL: "O",
```
Why save three bytes? Better readability for the win!